From ea9b38d4e1772a183e501f0162d6a1257af39c33 Mon Sep 17 00:00:00 2001
From: Raj, Deepu <deepu.raj@ti.com>
Date: Wed, 12 Oct 2011 11:39:05 +0530
Subject: [PATCH 2/2]  TI814X : Fixed autoload boot.scr issue in SD boot

 Auto loading of boot.scr from SD card was not
 happening  due to mmc command change.This is
 fixed by updating the 2nd stage boot cmd with
 'mmc rescan 0'.

Signed-off-by: Raj, Deepu <deepu.raj@ti.com>
---
 include/configs/ti8148_evm.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/configs/ti8148_evm.h b/include/configs/ti8148_evm.h
index 5c9ed1e..cd0f56f 100644
--- a/include/configs/ti8148_evm.h
+++ b/include/configs/ti8148_evm.h
@@ -105,7 +105,7 @@
 		"source ${script_addr}\0" \
 
 # define CONFIG_BOOTCOMMAND \
-	"if mmc init; then " \
+	"if mmc rescan 0; then " \
 		"if run loadbootscript; then " \
 			"run bootscript; " \
 		"else " \
-- 
1.7.0.4

